### DISCUSSION TITLE
ST7789 320x240 ESP32 with SDCard and Touch

### DISCUSSION THREAD

**Discussion by Terryon5Star** [Bodmer/TFT_eSPI#2574] [Q&A - Touch]
Hi Bodmer, I am using your library to start with my display.
My problem is that I use a SPI display with a SDCard connector and a ST7789 driver.
I thought all along that I have an ILI9341 driver, so I also played a sketch on my ST7789 320x240 display, but I had an error spot which was also dynamic, so I didn't know what to do.

Now I found out that I have a ST7789 driver  with a SDCard slot and also touch funktionality, but I don't know how to connect it and enable touch or my SDCard slot.

If you wan't pictures I can give it to you

Discord: Trydener on 5$tar#1108
email: imade624@gmail.com

**Comment by Bodmer** [comment]
Provide a link to the seller pages of the processor and display.
Provide a zip copy of your setup file.

**Reply by Bodmer** [reply]
@Terryon5Star 
Please only post on this topic here and not on other topics. I have deleted the incorect postings.
